x-pack/plugins/infra/public/pages/metrics/hosts_landing/index.tsx

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Just out of curiosity: could this component be in `hosts` root folder?

Reply:
I started it as a separate page (with a separate path) now as this page is rendered under the same route it can go to the hosts folder, I agree. I moved it together with the other content and removed the folder.